test/thermo/thermoToYaml.cpp

```diff
@@ -517,7 +517,6 @@ TEST_F(ThermoYamlRoundTrip, Surface)
 TEST_F(ThermoYamlRoundTrip, IsotropicElectronEnergyPlasma)
 {
     roundtrip("oxygen-plasma.yaml", "isotropic-electron-energy-plasma");
-    compareThermo(800, 2*OneAtm);
     auto origPlasma = std::dynamic_pointer_cast<PlasmaPhase>(original);
```

Comment:
Why does this comparison have to be removed?

Reply:
Mixture thermal properties like enthalpy_mole() are not applicable for PlasmaPhase because it has gas and electron temperature.

Reply:
The initial point of this PR was to provide a working implementation of `enthalpy_mole`, though. I pushed an alternative that selectively disables the two methods called by `compareThermo` that are not implemented in `PlasmaPhase` (`cp_mole` and `entropy_mole`).